src/Models/FailedJob.php

```diff
@@ -52,16 +52,16 @@ public function getDisplayNameAttribute(): string
 	/**
 	 * Get value of maxTries attribute.
 	 */
-	public function getMaxTriesAttribute(): int
+	public function getMaxTriesAttribute(): ?int
 	{
 		return $this->payload['maxTries'];
```

Comment:
Any reason it's not `return $this->payload['maxTries'] ?? null;` as in `getDelayAttribute`?
For consistency I would prefer to add it if you didn't see any errors because of that.